crates/ruff/src/rules/wemake_python_styleguide/helpers/naming.rs

```diff
@@ -0,0 +1,70 @@
+use std::path::Path;
+
+const ALIAS_NAMES_ALLOWLIST: [&str; 7] = ["np", "pd", "df", "plt", "sns", "tf", "cv"];
+const UNUSED_PLACEHOLDER: char = '_';
+
+/// Checks whether the given ``name`` is unused.
+fn is_unused(name: &str) -> bool {
+    name.chars().all(|c| c == UNUSED_PLACEHOLDER)
+}
+
+/// Checks for too short names.
+pub(in crate::rules::wemake_python_styleguide) fn is_too_short_name(
+    name: &str,
+    min_length: usize,
+    trim: bool,
+) -> bool {
+    if ALIAS_NAMES_ALLOWLIST.contains(&name) {
+        return false;
+    }
+
+    if is_unused(name) {
+        return false;
+    }
+
+    let length = if trim {
+        name.trim_matches(UNUSED_PLACEHOLDER).len()
+    } else {
+        name.len()
```

Comment:
Len gives you the byte length of a string and not the character length. You have to do `name.chars().count()` to get the character count instead.

Reply:
A common gotcha. I always forget that unicode strings are so special.